protos/feast/core/CoreService.proto

```diff
@@ -215,4 +220,39 @@ message ListProjectsRequest {
 message ListProjectsResponse {
     // List of project names (archived projects are filtered out)
     repeated string projects = 1;
-}
\ No newline at end of file
+}
+
+message GetFeatureStatisticsRequest {
+    // Optional filter which filters returned statistics by selected features. These
+    // features must be present in the data that is being processed.
+    repeated string feature_refs = 1;
+
+    // Optional filter to select the stores over which the statistics will retrieved.
+    // Only historical stores are allowed.
+    repeated string stores = 2;
```

Comment:
Ok I will run with this. What is the use case? Is anyone asking for this?

Reply:
Although it's a rare usecase, we don't have an explicit restriction for only having a single historical store, so it is possible, and the code should account for that. We can do without this filter, but that means that we return results over all stores always.

Reply:
This API will only be used for batch/historical data. What we gain by adding this filter is for the 1% of people (engineers?) to be able to run statistics on remote stores, but in almost all other cases users will only care about the central warehouse, not a remote warehouse. 

Adding this field doesn't seem to add value, yet constrains our design in that GetStatistics cannot be moved to serving.

Can we punt this decision for now by electing a single store from configuration in core that will be used for both scheduled and on-demand statistics?

Reply:
IMO electing a single store for generation of statistics doesn't seem natural unless Feast has a clear concept of having a central warehouse store

Reply:
I agree that it is unnatural, since we are neither committing to the full "master core" approach or "serving first" approach. It's kind of an inbetween, unfortunately.

Reply:
But at the same time I do think its the safer choice right now.

Reply:
The compromise doesn't really sit well with me even if its easier for us to implement because it makes the system's behavior unintuitive :/

Reply:
It's not about ease of implementation, it's about reversibility. We can't move statistic generation to serving if we have this  API, and we also can't move to a model where a Feast deployment has a primary warehouse. One of which I think we should move to in the future. 

This buys us time.

Conversely it is trivial to add the `stores` field at any point in time and allow users to query statistics from remote stores.

Reply:
If you really have to implement this then I would suggest we allow only a single store.

Reply:
ok